nin/backend/nin

```diff
@@ -63,6 +64,29 @@ program
     generate.generate(projectRoot, resource, name);
   });
 
+program
+  .command('update')
+  .description('Update nin to newest version')
+  .action(function() {
+    echo('Checking for new version of nin');
```

Comment:
I find it confusing to suddenly use echo here when we use console.log for the same thing everywhere else